query/src/main/java/org/infinispan/query/clustered/QueryResponse.java

```diff
@@ -45,6 +45,17 @@ public QueryResponse(NodeTopDocs nodeTopDocs, UUID nodeUUid, int resultSize) {
       this.resultSize = resultSize;
    }
 
+   public boolean nonEmpty() {
```

Comment:
Just a matter of style, my feeling is we do not need this method since ```getResultSize``` can be used as well.